python-checks/src/main/java/org/sonar/python/checks/CheckUtils.java

```diff
@@ -113,4 +120,25 @@ public static boolean isCalledInClassBody(Symbol symbol, ClassDef classDef) {
     });
   }
 
+  public static Optional<FunctionSymbol> getOverriddenMethod(FunctionSymbol functionSymbol) {
```

Comment:
I'd suggest moving this method to `SymbolUtils` or `TreeUtils` in python-frontend so you don't need to declare Mockito dependency

Reply:
Agreed, I think it would make sense to have it in `SymbolUtils`. Also you wouldn't have to redefine the `pythonFile` helper method that way.